tpm_core_node:
  ros__parameters:
    alm_logic: 0    # 0:low active, 1:high active
    org_logic: 1    # 0:low active, 1:high active
    feedback_src: 0 # 0:external. 1:internal
    pulse_per_deg: [905.29,
                1142.8,
                1123.17,
                877.38,
                206.1,
                439.37]
    max_axes_jog_speed: [40.0, 20.0, 20.0, 40.0, 40.0, 40.0] #unit:deg
    home_mode: 1
    home_offsets: [-171.1, -39.2, 63.3, -150.8, -94.5, 145.3] #unit:deg
    home_dir: [0, 0, 1, 0, 0, 1] 
    #home_vel: [2000.0, 2000.0, 2000.0, 2000.0, 1000.0, 5000.0] #unit:pulse/s  #todo: shoule be deb/s
    